pkg/rpc/client/rpc.go

```diff
@@ -61,7 +61,7 @@ func (c *Client) GetApplicationLog(hash util.Uint256, trig *trigger.Type) (*resu
 // GetBestBlockHash returns the hash of the tallest block in the blockchain.
 func (c *Client) GetBestBlockHash() (util.Uint256, error) {
 	var resp = util.Uint256{}
-	if err := c.performRequest("getbestblockhash", request.NewRawParams(), &resp); err != nil {
+	if err := c.performRequest("getbestblockhash", nil, &resp); err != nil {
```

Comment:
To be compatible with C# server and until https://github.com/neo-project/neo-modules/issues/742 is resolved, can we keep an old behaviour and pass an empty array if empty interfaces instead of `nil`?